packages/dashboard-frontend/src/services/backend-client/kubernetesNamespaceApi.ts

```diff
@@ -24,7 +24,9 @@ export async function getKubernetesNamespace(): Promise<che.KubernetesNamespace[
 }
 
 export async function provisionKubernetesNamespace(): Promise<che.KubernetesNamespace> {
-  const response = await axios.post(`${cheServerPrefix}/kubernetes/namespace/provision`);
```

Comment:
There is an unused axios import at the top of the page.

Reply:
Removing this unsued import will fix the build.

Reply:
Thank you, I've made a new commit to remove it